Library/Formula/vile.rb

```diff
@@ -0,0 +1,23 @@
+class Vile < Formula
+  desc "Vi Like Emacs"
+  homepage "http://invisible-island.net/vile/vile.html"
+  url "http://invisible-island.net/datafiles/release/vile.tar.gz"
```

Comment:
It is stable, and the version is inside the tarball (i.e. when extracted, the version shows in the extracted folder). Unfortunately, that's how the author of the software hosts it and I can't change that.